lib/slack-ruby-bot/rspec/support/slack_ruby_bot_configure.rb

```diff
@@ -1,9 +1,15 @@
 RSpec.configure do |config|
   config.before :each do
+    allow(Giphy).to receive(:random) if ENV.key?('WITH_GIPHY')
     SlackRubyBot.configure do |c|
       c.token = 'testtoken'
       c.user = 'rubybot'
       c.user_id = 'DEADBEEF'
     end
   end
+
+  config.after :each do
+    ENV.delete 'SLACK_RUBY_BOT_SEND_GIFS'
```

Comment:
This doesn't look right. If we're not setting it before in this context we shouldn't be deleting it after.